runtime/rasdump/javadump.cpp

```diff
@@ -1484,6 +1484,87 @@ JavaCoreDumpWriter::writeEnvUserArgsHelper(J9VMInitArgs *vmArgs)
 				_OutputStream.writeCharacters("\n");
 			}
 		}
+
+		/* Ignored options for -Xjit and -Xaot follows the following rules:
+		 * 1. If Xjit is followed by -Xint or -Xnojit,
+			then any -Xjit:optionString beforehand appears as ignored
+		 * 2. If XX:+MergeCompilerOptions is present,
+			then all Xjit options after Xint or Xnojit should not be ignored
+		 * 3. If multiple Xjit string appear, all but the last one will appear as ignored
+		 *
+		 * and the same for -Xaot
+		 */
+
+		int numXjit = 0, numXaot = 0;
```

Comment:
As far as I know, each defined variable should stay on its own line.
Also see below: "int ignoreXjitBefore = 0, ignoreXaotBefore = 0;" and "int countXjit = 0, countXaot = 0;"